sample/package.json

```diff
@@ -31,7 +31,7 @@
 		"@types/mocha": "^8.2.0",
 		"@types/node": "^18",
 		"@types/vscode": "^1.52.0",
-		"glob": "^10.3.10",
+		"glob": "^9.0.0",
```

Comment:
Not sure how this slipped through before (yarn weirdness?) but Glob 10 only supports Node 20 and we verify our sample on Node 16 (our min supported node version)

Reply:
Oh, this was probably an unintentional change in a minor version of glob https://github.com/isaacs/node-glob/issues/594